tests/test_grad.py

```diff
@@ -2,7 +2,8 @@
 
 from grl import load_spec, do_grad, RTOL
 
-def test_example_7_p():
```

Comment:
This renaming is probably going to prevent pytest from automatically discovering the tests